soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_item.c

```diff
@@ -9,6 +9,7 @@ u8 gAmmoItems[] = {
 static s16 sEquipState = 0;
 static s16 sEquipAnimTimer = 0;
 static s16 sEquipMoveTimer = 10;
+bool gMaskSelect;
```

Comment:
having this use the same name as the cvar for enabling/disabling mask selection is confusing to me

i think it'd be more clear as just 
```cpp
bool selectingMask;
```

Reply:
Definitely more descriptive and clear than `gMaskSelect`. Implemented (with the `g` per below). 👍